analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.Vsix.csproj

```diff
@@ -64,9 +64,6 @@
     <None Include="SonarAnalyzer.VisualBasic.nuspec">
```

Comment:
this didn't get propagated to the VSIX proj - maybe link it to analyzers/src/Directory.Build.props ?

Reply:
It seems to be a bug in visual studio when displaying the link: https://developercommunity.visualstudio.com/t/File-links-added-in-DirectoryBuildprop/1456131?entry=problem

Actually the file is considered during build (we can easily test that by adding a duplicate attribute).